src/Microsoft.Identity.Client/Logger.cs

```diff
@@ -204,14 +204,16 @@ internal void VerbosePii(string message)
 
         /// <summary>
         /// Method for error exception logging
+        /// Removes Pii from exception
```

Comment:
the formatted message should also be updated to reflect if the log statement is PII or not (it is not already being done).

Reply:
I added "true/false" to the end of the log message so it will be easier to find (versus putting it in the middle). if you want it somewhere else, let me know.